.github/workflows/build-docs.yml

```diff
@@ -36,42 +35,73 @@ defaults:
     shell: bash -l {0}
 
 jobs:
-  build-docs:
+  Check-for-changes:
```

Comment:
```suggestion
  check-for-changes:
```

Reply:
I have renamed the job using lower case.